cumulusci/core/flowrunner.py

```diff
@@ -349,42 +349,46 @@ def run(self, org_config):
 
         try:
             for step in self.steps:
-                if step.skip:
-                    self._rule(fill="*")
-                    self.logger.info(f"Skipping task: {step.task_name}")
-                    self._rule(fill="*", new_line=True)
-                    continue
-
-                if step.when:
-                    jinja2_context = {
-                        "project_config": step.project_config,
-                        "org_config": self.org_config,
-                    }
-                    expr = jinja2_env.compile_expression(step.when)
-                    value = expr(**jinja2_context)
-                    if not value:
-                        self.logger.info(
-                            f"Skipping task {step.task_name} (skipped unless {step.when})"
-                        )
-                        continue
-
-                self._rule(fill="-")
-                self.logger.info(f"Running task: {step.task_name}")
-                self._rule(fill="-", new_line=True)
-
-                self.callbacks.pre_task(step)
-                result = TaskRunner.from_flow(self, step).run_step()
-                self.callbacks.post_task(step, result)
-
-                self.results.append(
-                    result
-                )  # add even a failed result to the result set for the post flow
-
-                if result.exception and not step.allow_failure:
-                    raise result.exception  # PY3: raise an exception type we control *from* this exception instead?
+                self._run_step(step)
+            self.logger.info(f"Completed flow '{self.name}' successfully! 🎉")
```

Comment:
- I would avoid saying "successfully" -- we know the flow completed without an exception, but we don't know if it did what the user intended.
- self.name may be None
- I would avoid the emoji unless you've tested to make sure it works okay in Windows and when the logs are stored in Django models and rendered in a web browser.